react/features/app/components/App.web.js

```diff
@@ -6,6 +6,7 @@ import React from 'react';
 import { DialogContainer } from '../../base/dialog';
 import '../../base/responsive-ui';
 import '../../chat';
+import '../../follow-me';
```

Comment:
Just a question: Why don't we use the abstract class to load platform independent dependencies? I know it's not related to this PR, just thinking

Reply:
overlooked it.